Project 3/style.css

```diff
@@ -0,0 +1,237 @@
+* {
+  font-family: "Inter", sans-serif;
+  padding: 0;
+  box-sizing: border-box;
+  font-size: 15px;
+}
+body {
+  height: 100vh;
```

Comment:
a czemu tak? Twój projekt co prawda wygląda dobrze, ale kontent wychodzi poza body :) to ograniczenie jest raczej zbędne, a może zaszkodzić